### DISCUSSION TITLE
HDMI-CEC not working

### DISCUSSION THREAD

**Discussion by noxish** [guysoft/FullPageOS#619] [Q&A]
Hi, 

i tried to turn on and off my TV with "cec-client"
but it ends with an error 

anyone has experience with libcec on FullpageOS?

**Comment by guysoft** [comment]
What are you controlling with CEC on a browser?

**Comment by noxish** [comment]
i want to control the TV with cec 

i created a script wich should turn the TV on and off while a special event triggers

**Comment by guysoft** [comment]
Can you share the script? Others won't be able to test it otherwise.
Will also note this is outside the scope of FullPageOS IMHO.

**Comment by xvilo** [comment]
Would be great if this would be within the scope of FullPageOS

**Comment by guysoft** [comment]
You are welcome to send a pull request. I have no time to implement this.

**Comment by Alternativend** [comment]
Any progress on this?

**Comment by guysoft** [comment]
No, as I said I have no time to develop it, and no one else tried to get it working.

@Alternativend You can play around with RC1 and read guides and report progress if you like. It might help us understand what is needed.

**Comment by sambartle** [comment]
This issue is still open.. however cec-utils seems to work fine for me today with the latest releases.

I'm using the Pi with fullpageos now to make TVs turn on and display webpages.. I just control cec-client using an @reboot crontab entry and when the pi starts, it turns on the TV and sets the right AV input.

`@reboot echo 'as 0'|cec-client -s -d 1`

**Comment by guysoft** [comment]
Its worth adding that to the wiki:
https://github.com/guysoft/FullPageOS/wiki

**Comment by peteruithoven** [comment]
I've added a relevant guide: 
https://github.com/guysoft/FullPageOS/wiki/Control-TV-over-HDMI
Feedback is welcome!

**Comment by guysoft** [comment]
@peteruithoven thanks for adding that.
I am converting this to a discussion.
